pkg/cmd/datastore/datastore.go

```diff
@@ -236,6 +243,21 @@ func newSpannerDatastore(opts Config) (datastore.Datastore, error) {
 	)
 }
 
+func newMySQLDatastore(opts Config) (datastore.Datastore, error) {
+	mysqlOpts := []mysql.Option{
+		mysql.GCInterval(opts.GCInterval),
+		mysql.GCWindow(opts.GCWindow),
+		mysql.GCInterval(opts.GCInterval),
+		mysql.ConnMaxIdleTime(opts.MaxIdleTime),
+		mysql.ConnMaxLifetime(opts.MaxLifetime),
+		mysql.MaxOpenConns(opts.MaxOpenConns),
+		mysql.RevisionFuzzingTimedelta(opts.RevisionQuantization),
+		mysql.TablePrefix(opts.TablePrefix),
+		mysql.EnablePrometheusStats(),
```

Comment:
@sbryant I ended up enabling by default the prometheus stats, I couldn't figure out how to wire it up with `EnableDatastoreMetrics`, [just like the PSQL driver recently did](https://github.com/vroldanbet/spicedb/blob/0134a627bc5827c7e9929c1cf7e58c2e4289dd58/pkg/cmd/datastore/datastore.go#L222-L224).

@ecordell since you introduced that internal flag, is that something that can be passed as an ENV variable via CLI? I couldn't figure out why it wasn't being parsed.

Reply:
The way I read to cobrautil setup is that a flag still needs to be defined for the env var approach to work? https://github.com/jzelinskie/cobrautil/blob/main/cobrautil.go#L38

Reply:
Correct - right now that option is not bound to a cli flag. If you bind `EnableDatastoreMetrics` to a flag in `RegisterDatastoreFlags` then you'll be able to set it via CLI or env vars

Reply:
@ecordell how does that flag even work then? How are metrics ever turned on?

Reply:
@authzed test ping

Reply:
@sbryant It wasn't being used anywhere, it was a flag we could turn on during tests if we wanted to (via code)

Reply:
> @sbryant It wasn't being used anywhere, it was a flag we could turn on during tests if we wanted to (via code)

Ahh okay! Thanks for the explanation

Reply:
@ecordell as it stands, it means that postgresql prometheus metrics are off by default, and there is no way to turn them on via the CLI? Is that intentional? They used to be _on_ by default so that would be a regression for any customers running PSQL

Reply:
Sounds like this may need to be addressed, but a followup should do

Reply:
@vroldanbet You're right, they were on by default. I've filed https://github.com/authzed/spicedb/issues/549 to follow up and change the default